dev/breeze/doc/05_test_commands.rst

```diff
@@ -317,6 +317,25 @@ through ``breeze testing docker-compose-tests`` command.
 
 The docker-compose tests are in ``docker-tests/`` folder in the main repo.
 
+Running task-sdk integration tests
+..................................
+
+You can use Breeze to run the task sdk integration tests. Those tests are run using Production image by default
+and the tests are running with the docker-compose we have for task-sdk tests.
+
+.. image:: ./images/output_testing_docker-compose-tests.svg
+  :target: https://raw.githubusercontent.com/apache/airflow/main/dev/breeze/images/output_testing_task-sdk-integration-tests.svg
+  :width: 100%
+  :alt: Breeze testing task-sdk-integration-tests
+
+You can also iterate over those tests with pytest command but unlike regular unit tests, they need to be run in
+a local venv. You can build the prod image with breeze and that will be used by default if present to run the tests.
+
+You can override the ``DOCKER_IMAGE`` environment variable to point to the image to test using the
+``breeze testing task-sdk-integration-tests`` command.
```

Comment:
These tests are also backported and now this is a backport of the docs?

Reply:
The "tool" is backported - but I have not backported the tests themselves for task-sdk. But making the changes in breeze to be "on-par" with main, allows to do it  actually (but I wanted to limit this PR to only synchronize "dev/ci" framework - but not necessarliy tests themselves.

I also don't think it's strictly needed in this case - it would be great to actually do that. Those could be relatively easy backported - I think, but I will leave it to @amoghrajesh .